src/main/java/com/example/zoospringv1/ticketOffice/Ticket.java

```diff
@@ -0,0 +1,16 @@
+package com.example.zoospringv1.ticketOffice;
+
+public class Ticket {
+    int numberOfTickets;
```

Comment:
czy wiesz czym jest enkapsulacja?
czy wiesz czym są klasyfikatory dostępu?
jaki powinno mieć to pole klasy, aby być prawidłowo zenkapsulowane?